packages/atomic/package.json

```diff
@@ -34,7 +34,12 @@
     "./lang/*": {
       "import": "./dist/atomic/lang/*",
       "require": "./dist/atomic/lang/*"
-    }
+    },
+    "./components/atomic-text": {
```

Comment:
We'll probably need this for the lazy-loading.
- [ ] TODO: Generate export map from CEM.